.storybook/recipes/GlobalHeader/GlobalHeader.tsx

```diff
@@ -1,24 +1,19 @@
 import clsx from 'clsx';
 import React, { useState, useEffect } from 'react';
 import styles from './GlobalHeader.module.css';
-import {
-  Header,
-  Logo,
-  PrimaryNav,
-  PrimaryNavItem,
-  NavContainer,
-  Button,
-  Avatar,
-  UtilityNav,
-  UtilityNavItem,
-  Icon,
-  Popover,
-  PopoverBody,
-  PopoverHeader,
-  Heading,
-  NotificationList,
-  NotificationListItem,
-} from '../../../src';
+import { Button, Icon, Heading } from '../../../src';
+// Project Overview pilot components -- not yet exported from src/index.ts
+import Avatar from '../../../src/components/Avatar';
+import Header from '../../../src/components/Header';
+import Logo from '../../../src/components/Logo';
+import NavContainer from '../../../src/components/NavContainer';
+import NotificationList from '../../../src/components/NotificationList';
```

Comment:
Notifications are in the list for the CAD project (though I don't know if that includes this component specifically).

Reply:
oh interesting ... I _think_ that's Inline Notifications & Banners and such, whereas this is specifically the Notifications Popover in the left nav footer

Reply:
Ah, that makes sense, thanks!